extensions/sonarcloud/tasks/analyze/v1/task.json

```diff
@@ -11,14 +11,28 @@
   "author": "sonarsource",
   "version": {
     "Major": 1,
-    "Minor": 39,
+    "Minor": 40,
     "Patch": 0
   },
   "minimumAgentVersion": "2.144.0",
   "demands": [
     "java"
   ],
-  "inputs": [],
+  "inputs": [
+    {
+      "name": "jdkversion",
+      "type": "pickList",
+      "label": "JDK version source for analysis",
+      "defaultValue": "JAVA_HOME_11_X64",
+      "required": true,
+      "options": {
+          "JAVA_HOME": "Use built-in or provide JAVA_HOME",
+          "JAVA_HOME_11_X64": "Use built-in (hosted agent) JAVA_HOME_11_X64",
+          "JAVA_HOME_17_X64": "Use built-in (hosted agent) JAVA_HOME_17_X64"
```

Comment:
Nitpick

Using the assistant, the option choices are hidden by default. By hovering them it is still possible to have the tooltip, but may be better to re-organize the wording

![image](https://github.com/SonarSource/sonar-scanner-vsts/assets/31401273/0b4b22a3-a640-4308-abae-b4350af97d26)

Could we rephrase them so the java version is visible directly?
```suggestion
          "JAVA_HOME": "Use built-in or provide JAVA_HOME",
          "JAVA_HOME_11_X64": "Use built-in JAVA_HOME_11_X64 (hosted agent)",
          "JAVA_HOME_17_X64": "Use built-in JAVA_HOME_17_X64 (hosted agent)"
```